src/Fields/ArrayBlocks.jl

```diff
@@ -296,6 +296,14 @@ function lazy_map(::typeof(evaluate),a::LazyArray{<:Fill{<:BlockMap}},x::Abstrac
   lazy_map(k,args...)
 end
 
+function lazy_map(k::Broadcasting,a::LazyArray{<:Fill{<:BlockMap}})
```

Comment:
This is the only think that I am not sure about in this PR. 

Perhaps replacing `Broadcasting` by `BroadcastingFieldOpMap` makes more sense.

Reply:
> This is the only part that I am not sure about in this PR.

Ok, this is what I was also wondering; see https://github.com/gridap/Gridap.jl/issues/637#issuecomment-898387437

> Perhaps replacing Broadcasting by BroadcastingFieldOpMap makes more sense.

To put you in context (if you were not already), this `lazy_map` overload is currently triggered from this line

https://github.com/gridap/Gridap.jl/blob/master/src/ReferenceFEs/RaviartThomasRefFEs.jl#L392

If I understand correctly, you want to defer this "change of order" of operations till evaluation, right? Apart from consistency with other parts of the code, what are the benefit of this? (for my understanding).